src/pyhf/infer/calculators.py

```diff
@@ -597,7 +628,7 @@ def teststatistic(self, poi_test):
             Float: The value of the test statistic.
 
         """
-        teststat_func = qmu_tilde if self.qtilde else qmu
+        teststat_func = test_statistics.get(self.test_stat)
```

Comment:
Only applies if people agree on the API comment

```suggestion
        teststat_func = test_statistics.get_test_stat(self.test_stat)
```